iface.go

```diff
@@ -3,4 +3,11 @@ package celeriac
 // Broker defines the features which all drivers must implement
 type Broker interface {
 	Close() error
+	GetQueue(string) (Queue, error)
+}
+
+// Queue defines the features of a Queue which all drivers must implement
+type Queue interface {
+	Consume() (<-chan Message, error)
+	Publish([]byte) error
```

Comment:
I think we should return the `MessageId` after `Publish`. Reason being we may wish to pass the ID onto the caller so that the queue can be queried at a later date to check the progress of the task.

Reply:
done